### DISCUSSION TITLE
Help i can't access to file .php on web

### DISCUSSION THREAD

**Discussion by winddragonn** [phcode-dev/phoenix#1090] [Q&A]
I got "405 Not Allowed"
I had check all the code and It's has no bug
I'm a newbie and sorry if this question is so dump
Thanks, have a good day

**Comment by abose** [comment]
php requires php runtime. so it cannot live preview in phoenix directly.
Can you explain your workflow more?

**Comment by Alaskadadabra** [comment]
HI. I have the same issue. I do not use index.html but index.php to be able to use self-explanatory php codes within the file.
For example, to refer to html in the head area that is in the same directory.
<head>
    <?php include 'head.html'; ?>
</head>

I want several pages to access this head.html so that I don't have to change it for every page, but can only overwrite one html at a time.
I am also a beginner. am I doing something wrong in the workflow?

In any case, the problem is that I can no longer use live-view from phoenix-code.

kind regards

**Reply by abose** [reply]
For PHP, id recommended using http://brackets.io till we add support for php in phcode ina few months.

YouTube will have several videos for `php live preview in brackets`

https://youtu.be/H2LfvBYT1ic?si=lCCw_zxY2ByVHTvH

**Reply by Alaskadadabra** [reply]
great, thank you for the quick reply. looking forward to the php-live-view-function!
